options/locale/locale_en-US.ini

```
@@ -3220,6 +3220,7 @@ none = There are no secrets yet.
 value = Value
 name = Name
 creation = Add Secret
+creation.name_placeholder = Not case-sensitive, alphanumeric characters or underscores only.
```

```suggestion
creation.name_placeholder = case-insensitive, alphanumeric characters or underscores only, cannot start with GITEA_ or GITHUB_
```
(regarding the first change in this line: At least for me, `not case-sensitive` is 1000× more confusing than `case-insensitive`)